.github/workflows/js-ci.yaml

```diff
@@ -186,6 +186,9 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Set environment variables for testing purposes
+        run: echo "DOMAIN_NAME=test1.eth STAGE_NAME=stage PACKAGE_NAME=package RANDOM_NAME=name" >> $GITHUB_ENV
```

Comment:
@cbrzn  check this out: https://docs.github.com/en/actions/learn-github-actions/environment-variables